swri_transform_util/include/swri_transform_util/earth_constants.h

```diff
@@ -37,6 +37,9 @@ namespace swri_transform_util
    */
   static const double _earth_equator_radius = 6378137.0;
 
+  /**
+   * Earth mean radius
```

Comment:
I agree. What units is it in? I would assume meters, but I didn't write the constant.